Uno/NuGetPackageExplorer/Platforms/WebAssembly/wwwroot/staticwebapp.config.json

```diff
@@ -8,6 +8,13 @@
     ]
   },
   "routes": [
+    {
+      "route": "/_framework/*",
+      "headers": {
+        "cache-control": "public, immutable, max-age=31536000"
```

Comment:
Will this cause any caching issues or are the files deployed with unique path/querstring args?

Reply:
I don't think so. These files are never referenced as part of query strings, and are equivalent to the `/package_*` route below. This folder contains the dlls now (in a `.wasm` format).